### PR TITLE
test: collect stdout/stderr in tests

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

**Review by dgozman — APPROVED** [review]

File: test-runner/test/assets/stdio.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require('../../..');
+
+it('stdio', () => {
+  process.stdout.write('stdout text');
```
Comment (JoelEinbinder) [inline test-runner/test/assets/stdio.js:20]: Does this work with console logs? I'm scared about our method of overwriting process.stdout.

File: test-runner/test/assets/stdio.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  process.stdout.write(Buffer.from('stdout buffer'));
+  process.stderr.write('stderr text');
+  process.stderr.write(Buffer.from('stderr buffer'));
```
Comment (JoelEinbinder) [inline test-runner/test/assets/stdio.js:23]:
```suggestion
  process.stderr.write(Buffer.from('stderr buffer'));
  console.log('console log');
  console.error('console.error');
```

**Review by JoelEinbinder — APPROVED** [review]

File: test-runner/test/assets/stdio.js (hunk 1 above)
Comment (pavelfeldman) [reply test-runner/test/assets/stdio.js:20]: It does, i'm adding a test.

File: test-runner/test/assets/stdio.js (hunk 2 above)
Comment (pavelfeldman) [reply test-runner/test/assets/stdio.js:23]: Yup, added a separate test!

**Pull request merged**